src/features/projectsV2/ProjectsMap/microComponents/PlantLocations.tsx

```diff
@@ -34,6 +34,10 @@ export default function PlantLocations(): ReactElement {
   const locale = useLocale();
 
   const openPl = (pl: PlantLocationSingle | SamplePlantLocation) => {
+    if (selectedSamplePlantLocation?.hid === pl.hid) {
```

Comment:
This allows the user to unselect the sample plant location while clicking on it again. However, I feel the sample plant location should also get unselected if the user clicks on the surrounding plant location again (this does not currently happen)